python/grass/jupyter/Makefile

```diff
@@ -5,7 +5,7 @@ include $(MODULE_TOPDIR)/include/Make/Python.make
 
 DSTDIR = $(ETC)/python/grass/jupyter
 
-MODULES = setup display
+MODULES = setup display interact_display
```

Comment:
To (partially) future proof this, you can list each module (name) on a separate line. I just did that for the Makefile in parent dir in this PR: https://github.com/OSGeo/grass/pull/1715/files#diff-ca3c8a522b85886bddf093dff36a30011e88cd6c5335d554d2f1ffb08582903d